Model/FeatureSwitch.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+
+namespace Bolt\Boltpay\Model;
+
+use Magento\Framework\Model\AbstractModel;
+use Bolt\Boltpay\Api\Data\FeatureSwitchInterface;
+
+class FeatureSwitch extends AbstractModel implements \Bolt\Boltpay\Api\Data\FeatureSwitchInterface
+{
+    const NAME = 'switch_name';
+    const VALUE = 'switch_value';
+    const DEFAULT_VALUE = 'default_value';
+    const ROLLOUT_PERCENTAGE = 'rollout_percentage';
+
+    protected function _construct()
+    {
+        parent::_construct();
+        $this->_init(ResourceModel\FeatureSwitch::class);
+    }
+
+    public function getName()
```

Comment:
my understanding is you don't have to define these getters/setters because magento has magic method (see "Basic Model Operations" section https://devdocs.magento.com/guides/m1x/magefordev/mage-for-dev-5.html) - yay dynamic language

Reply:
it was pretty hard to get here though :P their documentation though rich didnt add a sample for for this

Reply:
ok i understand how this is supposed to work but also thoughts on making it explicit?

this will make the method named getSwitchName() etc, which is fine, but i sort of like this typing. going ot let you and @matt-thomason decide

Reply:
discussed offline, that was M1 and M2 doesnt support this.